### PR TITLE
Propagating change from Euroc challengers.

### PR DESCRIPTION
**Pull request by alexmillane**
Also removing some evil tabs ;). Builds OK.

### CODE REVIEW

**Comment by alexmillane** [comment]
@helenol

**Comment by helenol** [comment]
Hooray no more tabs. LGTM.

File: CMakeLists.txt [hunk 1]
```diff
@@ -13,9 +13,10 @@ ExternalProject_Add(vrpn_src
   UPDATE_COMMAND ""
   CONFIGURE_COMMAND cmake ../vrpn_src -DVRPN_BUILD_SERVERS:BOOL=OFF
                                       -DVRPN_BUILD_SERVER_LIBRARY:BOOL=OFF
-									  -DCMAKE_OSX_ARCHITECTURES:STRING=x86_64
-									  -DVRPN_BUILD_JAVA:BOOL=OFF
-									  -DVRPN_BUILD_PYTHON_HANDCODED_2X:BOOL=OFF
+                                      -DCMAKE_OSX_ARCHITECTURES:STRING=x86_64
+                                      -DVRPN_BUILD_JAVA:BOOL=OFF
+                                      -DVRPN_BUILD_PYTHON_HANDCODED_2X:BOOL=OFF
```
Comment (ffurrer) [inline CMakeLists.txt]: remove?

Comment (alexmillane) [inline CMakeLists.txt]: Hmmm. Ok. Mystery flag removed.

**Pull request merged**